src/async_monitored_item.rs

```diff
@@ -222,21 +217,50 @@ impl MonitoredItemBuilder {
     }
 }
 
+/// Value emitted from monitored item notification.
+///
+/// The variant depends on the attribute ID passed to [`MonitoredItemBuilder::attribute_id()`].
+#[derive(Debug)]
+pub enum MonitoredItemValue {
```

Comment:
According to the docs this could also be named more generically as `MonitoredItemNotification`. Currently it only includes the last argument from the notification callback as the payload.

@sgoll @MartinTheisTrumpf Any opinions?